### ISSUE TITLE
gate_io: Failing to cancel order - Order not found in inflight orders

### ISSUE THREAD

**Issue by RC-13** [CoinAlpha/hummingbot#4647]
**Describe the bug**
When running any market making strategy using gate_io connector, after running the bot for a while, it starts to unable to recognize that one of the previous order created was already been cancelled. This results for the bot to keep trying to find the order and preventing the creation of new orders.
When checking the log, it shows that the bot never receive a completion event for the cancellation of orders, also the affected order appears to be an order that fails to be submitted because it doesn't meet the exchange minimum allowable trade size.

```python
2021-11-04 00:18:01,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Creating 3 bid orders at (Size, Price): ['0.002 ETH, 4585.07 USDT', '0.002 ETH, 4585.03 USDT', '0.002 ETH, 4584.98 USDT']
2021-11-04 00:18:01,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Creating 2 ask orders at (Size, Price): ['0.0019 ETH, 4619.95 USDT', '0.0009 ETH, 4619.96 USDT']
2021-11-04 00:18:01,006 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Sell order amount 0.0009 is lower than the minimum order size 0.001.
2021-11-04 00:18:01,116 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT SELL order t-HBOT-S-EHUT1635985081002832 for 0.0019 ETH-USDT.
2021-11-04 00:18:01,125 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985081.0, "type": "OrderType.LIMIT", "trading_pair": "ETH-USDT", "amount": "0.0019", "price": "4619.95", "order_id": "t-HBOT-S-EHUT1635985081002832", "exchange_order_id": "90331327439", "leverage": 1, "position": "NILL", "event_name": "SellOrderCreatedEvent", "event_source": "gate_io"}
2021-11-04 00:18:01,126 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT BUY order t-HBOT-B-EHUT1635985081002585 for 0.0020 ETH-USDT.
2021-11-04 00:18:01,135 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985081.0, "type": "OrderType.LIMIT", "trading_pair": "ETH-USDT", "amount": "0.0020", "price": "4585.03", "order_id": "t-HBOT-B-EHUT1635985081002585", "exchange_order_id": "90331327459", "leverage": 1, "position": "NILL", "event_name": "BuyOrderCreatedEvent", "event_source": "gate_io"}
2021-11-04 00:18:01,136 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT BUY order t-HBOT-B-EHUT1635985081002635 for 0.0020 ETH-USDT.
2021-11-04 00:18:01,145 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985081.0, "type": "OrderType.LIMIT", "trading_pair": "ETH-USDT", "amount": "0.0020", "price": "4584.98", "order_id": "t-HBOT-B-EHUT1635985081002635", "exchange_order_id": "90331327442", "leverage": 1, "position": "NILL", "event_name": "BuyOrderCreatedEvent", "event_source": "gate_io"}
2021-11-04 00:18:01,146 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT BUY order t-HBOT-B-EHUT1635985081002472 for 0.0020 ETH-USDT.
2021-11-04 00:18:01,155 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985081.0, "type": "OrderType.LIMIT", "trading_pair": "ETH-USDT", "amount": "0.0020", "price": "4585.07", "order_id": "t-HBOT-B-EHUT1635985081002472", "exchange_order_id": "90331327489", "leverage": 1, "position": "NILL", "event_name": "BuyOrderCreatedEvent", "event_source": "gate_io"}
2021-11-04 00:18:06,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-B-EHUT1635985081002472. [clock=2021-11-04 00:18:06+00:00]
2021-11-04 00:18:06,003 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-B-EHUT1635985081002585. [clock=2021-11-04 00:18:06+00:00]
2021-11-04 00:18:06,003 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-B-EHUT1635985081002635. [clock=2021-11-04 00:18:06+00:00]
2021-11-04 00:18:06,003 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-S-EHUT1635985081002832. [clock=2021-11-04 00:18:06+00:00]
2021-11-04 00:18:06,003 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-S-EHUT1635985081002889. [clock=2021-11-04 00:18:06+00:00]
2021-11-04 00:18:06,008 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel order t-HBOT-S-EHUT1635985081002889. Order not found in inflight orders.
2021-11-04 00:18:06,009 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-S-EHUT1635985081002889: (no details available)
NoneType: None
2021-11-04 00:18:06,009 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel the order t-HBOT-S-EHUT1635985081002889 on gate_io. Check API key and network connection.
2021-11-04 00:18:06,094 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-S-EHUT1635985081002832 on gate_io.
2021-11-04 00:18:06,104 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985086.0, "order_id": "t-HBOT-S-EHUT1635985081002832", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-04 00:18:06,105 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-B-EHUT1635985081002585 on gate_io.
2021-11-04 00:18:06,115 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985086.0, "order_id": "t-HBOT-B-EHUT1635985081002585", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-04 00:18:06,115 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-B-EHUT1635985081002635 on gate_io.
2021-11-04 00:18:06,126 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985086.0, "order_id": "t-HBOT-B-EHUT1635985081002635", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-04 00:18:06,126 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-B-EHUT1635985081002472 on gate_io.
2021-11-04 00:18:06,136 - 28722 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1635985086.0, "order_id": "t-HBOT-B-EHUT1635985081002472", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-04 00:19:07,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-S-EHUT1635985081002889. [clock=2021-11-04 00:19:07+00:00]
2021-11-04 00:19:07,003 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel order t-HBOT-S-EHUT1635985081002889. Order not found in inflight orders.
2021-11-04 00:19:07,004 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-S-EHUT1635985081002889: (no details available)
NoneType: None
2021-11-04 00:19:07,004 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel the order t-HBOT-S-EHUT1635985081002889 on gate_io. Check API key and network connection.
2021-11-04 00:20:08,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-S-EHUT1635985081002889. [clock=2021-11-04 00:20:08+00:00]
2021-11-04 00:20:08,003 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel order t-HBOT-S-EHUT1635985081002889. Order not found in inflight orders.
2021-11-04 00:20:08,004 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-S-EHUT1635985081002889: (no details available)
NoneType: None
2021-11-04 00:20:08,004 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel the order t-HBOT-S-EHUT1635985081002889 on gate_io. Check API key and network connection.
2021-11-04 00:21:09,002 - 28722 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (ETH-USDT) Cancelling the limit order t-HBOT-S-EHUT1635985081002889. [clock=2021-11-04 00:21:09+00:00]
2021-11-04 00:21:09,003 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel order t-HBOT-S-EHUT1635985081002889. Order not found in inflight orders.
2021-11-04 00:21:09,004 - 28722 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-S-EHUT1635985081002889: (no details available)
NoneType: None
```

This behavior is so far observed when running with multiple orders on ETH-USDT market.

**Steps To Reproduce**

Here are the steps to reproduce the issue (see attachments in the section below):

1. Create a any market making strategy using Gate_io as connector
1. Trade on ETH-USDT market
1.  Start the strategy and monitor the logs

...

**Screenshots**
![image](https://user-images.githubusercontent.com/57189990/140245483-8a7c26c3-2d9f-4e71-88f9-e7125aa66bad.png)

**Release version**
0.44.0

**Attachments**
[config and logs.zip](https://github.com/CoinAlpha/hummingbot/files/7465509/config.and.logs.zip)

_WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!_

**Optional: your discord username:**

┆Issue is synchronized with this [Clickup task](https://app.clickup.com/t/1uf6m94) by [Unito](https://www.unito.io)

**Comment by gerald-hb** [comment]
Discord user: belchi0r#6712 experienced the same error. He is running ETH-USDT pair in gate.io using Avellaneda strategy. Config and logs attached

[confavellaneda.txt](https://github.com/CoinAlpha/hummingbot/files/7465714/confavellaneda.txt)
[logs_conf_avellaneda_market_making_2_-_Copy (1).log](https://github.com/CoinAlpha/hummingbot/files/7465715/logs_conf_avellaneda_market_making_2_-_Copy.1.log)

**Comment by rapcmia** [comment]
Got the same error when testing a single level on Pure market-making on Gate_io. The Testbot was set up to create only ask orders and got a log warning of `Buy order amount 0.0001 is lower than the minimum order size 0.001.` which is expected however when checking the `status` command, the bot had bid order under `0.0001` but none on Gate_io portal. When reaching the set refresh time, the bot then returns an failure to cancel the bid order `...Order not found in inflight orders`

On the screenshot below, the bot created 2 orders when the balance can only cater ask side:
![inflight errorr](https://user-images.githubusercontent.com/73840223/140306056-48851349-974a-4088-8fd8-1d7f35329fc3.png)

The bot fails to cancel the non-existing bid order:
![inflight error 2](https://user-images.githubusercontent.com/73840223/140306286-3d2f6d6d-4635-43b4-8d58-26cc22d7e8f8.png)

This also happens on Avellaneda:
![inflight error 2 dev-branch avemm](https://user-images.githubusercontent.com/73840223/140306414-e7bb8aa9-14bc-49d6-beaf-11c3da41ed09.png)

Steps to reproduce:
1. Create any market-making strategy
2. Set your balance that will create only one side of the order book
3. Start the bot, you would notice that it will still create orders
4. Upon cancellation of the order, the non-existing order has its own order_id

Attached are config and log files for PMM:
[logs_pmm-gateio-inflight-error.zip](https://github.com/CoinAlpha/hummingbot/files/7474487/logs_pmm-gateio-inflight-error.zip)

**Comment by RC-13** [comment]
Similar behavior is now seen on other markets, in this example the affected market is DOGE-USDT however instead of unable to find the order in the inflight order, this time the error is either `Order not tracked.` or `Order not found`, leaving the orders stale in the exchange.
```python
2021-11-05 05:23:16,542 - 19603 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 bid orders at (Size, Price): ['1.65 DOGE, 0.257952 USDT']
2021-11-05 05:23:16,543 - 19603 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 ask orders at (Size, Price): ['7.87 DOGE, 0.264477 USDT']
2021-11-05 05:23:18,187 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Error submitting BUY LIMIT order to gate_io for 1.65 DOGE-USDT 0.257952 - Your order size 1.65 is too small. The minimum is 1 USDT.
Traceback (most recent call last):
  File "/home/ubuntu/gate_io/hummingbot/connector/exchange/gate_io/gate_io_exchange.py", line 478, in _create_order
    order_result = await self._api_request("POST", CONSTANTS.ORDER_CREATE_PATH_URL, api_params, True)
  File "/home/ubuntu/gate_io/hummingbot/connector/exchange/gate_io/gate_io_exchange.py", line 380, in _api_request
    raise GateIoAPIError(parsed_response)
hummingbot.connector.exchange.gate_io.gate_io_utils.GateIoAPIError: {'label': 'INVALID_PARAM_VALUE', 'message': 'Your order size 1.65 is too small. The minimum is 1 USDT'}
2021-11-05 05:23:18,188 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Error submitting order to gate_io - Your order size 1.65 is too small. The minimum is 1 USDT.
2021-11-05 05:23:18,239 - 19603 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636089797.0, "order_id": "t-HBOT-B-DEUT1636089796543023", "order_type": "OrderType.LIMIT", "event_name": "MarketOrderFailureEvent", "event_source": "gate_io"}
2021-11-05 05:23:18,240 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT SELL order t-HBOT-S-DEUT1636089796543273 for 7.87 DOGE-USDT.
2021-11-05 05:23:18,495 - 19603 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636089797.0, "type": "OrderType.LIMIT", "trading_pair": "DOGE-USDT", "amount": "7.87", "price": "0.264477", "order_id": "t-HBOT-S-DEUT1636089796543273", "exchange_order_id": "90707406181", "leverage": 1, "position": "NILL", "event_name": "SellOrderCreatedEvent", "event_source": "gate_io"}
2021-11-05 05:23:21,679 - 19603 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Cancelling the limit order t-HBOT-S-DEUT1636089796543273. [clock=2021-11-05 05:23:21+00:00]
2021-11-05 05:23:21,679 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-B-DEUT1636089789521808: Order not tracked.
NoneType: None
2021-11-05 05:23:21,679 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel the order t-HBOT-B-DEUT1636089789521808 on gate_io. Check API key and network connection.
2021-11-05 05:23:23,424 - 19603 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-S-DEUT1636089796543273 on gate_io.
```
```python
2021-11-05 03:21:53,007 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 bid orders at (Size, Price): ['2E+1 DOGE, 0.257792 USDT']
2021-11-05 03:21:53,007 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 ask orders at (Size, Price): ['12.74 DOGE, 0.264318 USDT']
2021-11-05 03:21:53,086 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT SELL order t-HBOT-S-DEUT1636082513007799 for 12.74 DOGE-USDT.
2021-11-05 03:21:53,099 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082513.0, "type": "OrderType.LIMIT", "trading_pair": "DOGE-USDT", "amount": "12.74", "price": "0.264318", "order_id": "t-HBOT-S-DEUT1636082513007799", "exchange_order_id": "90683219782", "leverage": 1, "position": "NILL", "event_name": "SellOrderCreatedEvent", "event_source": "gate_io"}
2021-11-05 03:21:53,100 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT BUY order t-HBOT-B-DEUT1636082513007526 for 20.00 DOGE-USDT.
2021-11-05 03:21:53,108 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082513.0, "type": "OrderType.LIMIT", "trading_pair": "DOGE-USDT", "amount": "20.00", "price": "0.257792", "order_id": "t-HBOT-B-DEUT1636082513007526", "exchange_order_id": "90683219791", "leverage": 1, "position": "NILL", "event_name": "BuyOrderCreatedEvent", "event_source": "gate_io"}
2021-11-05 03:22:23,118 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Cancelling the limit order t-HBOT-B-DEUT1636082513007526. [clock=2021-11-05 03:22:23+00:00]
2021-11-05 03:22:23,118 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Cancelling the limit order t-HBOT-S-DEUT1636082513007799. [clock=2021-11-05 03:22:23+00:00]
2021-11-05 03:22:23,375 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-B-DEUT1636082513007526 on gate_io.
2021-11-05 03:22:23,576 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082543.0, "order_id": "t-HBOT-B-DEUT1636082513007526", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-05 03:23:20,469 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Error fetching data from https://api.gateio.ws/api/v4/spot/orders/90683219782. HTTP status is None. Retrying in 4s.
2021-11-05 03:23:21,896 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_websocket - WARNING - The websocket connection was unexpectedly closed.
2021-11-05 03:23:23,450 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_api_order_book_data_source - INFO - Subscribed to ['DOGE-USDT'] orderbook data streams...
2021-11-05 03:23:23,450 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Successfully cancelled order t-HBOT-S-DEUT1636082513007799.
2021-11-05 03:23:23,595 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082603.0, "order_id": "t-HBOT-S-DEUT1636082513007799", "exchange_order_id": null, "event_name": "OrderCancelledEvent", "event_source": "gate_io"}
2021-11-05 03:23:25,637 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 bid orders at (Size, Price): ['2E+1 DOGE, 0.25792 USDT']
2021-11-05 03:23:25,637 - 16832 - hummingbot.strategy.avellaneda_market_making.avellaneda_market_making - INFO - (DOGE-USDT) Creating 1 ask orders at (Size, Price): ['12.75 DOGE, 0.264451 USDT']
2021-11-05 03:23:26,181 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - NETWORK - Failed to cancel order t-HBOT-S-DEUT1636082513007799: Order not found
NoneType: None
2021-11-05 03:23:26,181 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - WARNING - Failed to cancel the order t-HBOT-S-DEUT1636082513007799 on gate_io. Check API key and network connection.
2021-11-05 03:23:26,182 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT SELL order t-HBOT-S-DEUT1636082605637936 for 12.75 DOGE-USDT.
2021-11-05 03:23:26,239 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082604.0, "type": "OrderType.LIMIT", "trading_pair": "DOGE-USDT", "amount": "12.75", "price": "0.264451", "order_id": "t-HBOT-S-DEUT1636082605637936", "exchange_order_id": "90683519095", "leverage": 1, "position": "NILL", "event_name": "SellOrderCreatedEvent", "event_source": "gate_io"}
2021-11-05 03:23:26,240 - 16832 - hummingbot.connector.exchange.gate_io.gate_io_exchange - INFO - Created LIMIT BUY order t-HBOT-B-DEUT1636082605637683 for 20.00 DOGE-USDT.
2021-11-05 03:23:26,365 - 16832 - hummingbot.core.event.event_reporter - EVENT_LOG - {"timestamp": 1636082604.0, "type": "OrderType.LIMIT", "trading_pair": "DOGE-USDT", "amount": "20.00", "price": "0.257920", "order_id": "t-HBOT-B-DEUT1636082605637683", "exchange_order_id": "90683519100", "leverage": 1, "position": "NILL", "event_name": "BuyOrderCreatedEvent", "event_source": "gate_io"}
```
[logs_doge-usdt-avellaneda-mm.log.zip](https://github.com/CoinAlpha/hummingbot/files/7480875/logs_doge-usdt-avellaneda-mm.log.zip)

**Comment by dennisocana** [comment]
Fixed in version 0.46 release.
